integrations/amazon_bedrock/pyproject.toml

```diff
@@ -14,6 +14,7 @@ authors = [
   { name = "deepset GmbH", email = "info@deepset.ai" },
 ]
 classifiers = [
+  "License :: OSI Approved :: Apache Software License",
```

Comment:
Does it make sense to mention the version (v2.0) here as well?

Reply:
That wouldn't be standard, see https://pypi.org/classifiers/